workspace/test_configurations.go

```diff
@@ -0,0 +1,89 @@
+package workspace
+
+import (
+	"runtime"
+	"strings"
+)
+
+type TestConfiguration struct {
+	// The static portion of the test Command, which will be run for every test on this track. Examples include `cargo test` or `go test`.
+	// Might be empty if there are platform-specific versions
+	Command string
+
+	// Windows-specific test command. Mostly relevant for tests wrapped by shell invocations. Falls back to `Command` if we're not running windows or this is empty.
+	WindowsCommand string
+}
+
+func (c *TestConfiguration) GetTestCommand() (string, error) {
+	var cmd string
+	if runtime.GOOS == "windows" && c.WindowsCommand != "" {
+		cmd = c.WindowsCommand
+	} else {
+		cmd = c.Command
+	}
+
+	if strings.Contains(cmd, "{{test_files}}") {
+		testFiles, err := getTestFiles()
+		if err != nil {
+			return "", err
+		}
+		cmd = strings.ReplaceAll(cmd, "{{test_files}}", strings.Join(testFiles, " "))
+	}
+
+	return cmd, nil
+}
+
+var TestConfigurations = map[string]TestConfiguration{
+	"8th": {
+		Command:        "tester.sh",
+		WindowsCommand: "tester.bat",
+	},
+	"ballerina": {
+		Command: "bal test",
+	},
+	"c": {
+		Command: "make",
+	},
+	"cfml": {
+		Command: "box task run TestRunner",
+	},
+	"cobol": {
+		Command:        "test.sh",
+		WindowsCommand: "test.ps1",
```

Comment:
```suggestion
		Command:        "./test.sh",
		WindowsCommand: "pwsh test.ps1",
```

Reply:
I've updated, but was just copying from the docs. they may need to be updated: https://exercism.org/docs/tracks/cobol/tests#h-windows